src/main/java/frc/robot/commands/MagControl.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Magazine;
+
+/**
+ * Creates commands for the magazine to run.
+ */
+public class MagControl extends CommandBase {
```

Comment:
why not change to `public class MagControl extends ForceMag` then u only need to add the `finish()` function since the other functions are teh same